### PR TITLE
Various cleanups and test fixes

### PR DESCRIPTION
**Pull request by anehx**
* Use yarn instead of npm since npm has problems with git deps
* Update dependencies
* Remove ember-sy dependency and implemented sidenav in this project
* Use new ember-adcssy but only for vars and use static built adcssy
* Fix statistic and user tests
* Reconfigure postcss
* Remove obsolete yuidoc configuration
* Remove all the bower things :heart:
* Use node 8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,28 +30,18 @@ You will need the following things properly installed on your computer.
 
 * `git clone git@github.com/adfinis-sygroup/timed-frontend`
 * `cd timed-frontend`
-* `npm install`
-* `bower install`
-
-## Configurations
```
Comment (kaldras) [inline README.md]: Is the configuration part no longer needed/possible?

File: README.md [hunk 2]
```diff
@@ -30,28 +30,18 @@ You will need the following things properly installed on your computer.
...
 
 ## Running / Development
 
-* `ember serve`
+* `ember server`
 * Visit your app at [http://localhost:4200](http://localhost:4200).
 
 If you have a running [backend](https://github.com/adfinis-sygroup/timed-backend) you need to run
 
 * `ember serve --proxy=http://localhost:8000`
```
Comment (kaldras) [inline README.md]: Also change to `ember server` for consistency

File: README.md (hunk 1 above)
Comment (anehx) [reply README.md]: Yes, it was removed in #124

File: README.md (hunk 2 above)
Comment (anehx) [reply README.md]: Done

**Pull request merged**